### ISSUE TITLE
Doesn't continue training from checkpoint.

### ISSUE THREAD

**Issue by ailgun** [llSourcell/Pokemon_GAN#17]
Hello, 

I'm having a minor issue when trying to continue an interrupted training.

It is creating these files in /tmp/ as it's supposed to:
<img width="240" alt="screen shot 2018-08-05 at 7 54 12 pm" src="https://user-images.githubusercontent.com/31825138/43688019-f6c1869a-98e9-11e8-912b-d7a10399c597.png">
And these under model/NewPokemon in my working directory:
<img width="231" alt="screen shot 2018-08-05 at 7 54 56 pm" src="https://user-images.githubusercontent.com/31825138/43688029-0db93a46-98ea-11e8-9ccf-8b440d9defb2.png">

On other code I worked on, I would import the saved weights and biases with something like;

```
saver = tf.train.import_meta_graph('model.meta')
saver.restore(sess, tf.train.latest_checkpoint('./'))
```
On this code I just couldn't make it continue from where it is left off. It starts from 0 back again when I run PokeGAN.py even though I'm running inside the directory and the folder structure seems correct.

I haven't modified the code except the fact that, mine saves the model every epoch instead of every 500 - just changed the modulo i%500 to i%1.

Any help would be much appreciated.

Thanks very much!

**Comment by b3nh** [comment]
@ailgun did you ever resolve this problem and get training to continue after interruption? I'm facing similar issue

**Comment by HYPERCORTEX-CORP** [comment]
@arklabco Nope, sorry! I've decided to learn more by following a few tutorials online and creating a simple GAN but using NVIDIAs code for actual training :-)

**Comment by HYPERCORTEX-CORP** [comment]
Nope, sorry! I've decided to learn more by following a few tutorials online and creating a simple GAN but using NVIDIAs code for actual training :-)

**Comment by ailgun** [comment]
@arklabco  Nope, sorry! I've decided to learn more by following a few tutorials online and creating a simple GAN but using NVIDIAs code for actual training :-)

**Comment by orl0v** [comment]
Have encountered the same problem. Has anyone solved it?
